src/framework/parsers/glb-container-resource.js

```diff
@@ -123,7 +123,7 @@ class GlbContainerResource {
         // helper function to recursively clone a hierarchy of GraphNodes to Entities
         const cloneHierarchy = (root, node, glb) => {
 
-            const entity = new Entity();
+            const entity = new Entity('Untitled', this._assets._loader._app);
```

Comment:
just pass `undefined` instead of 'Untitled', let the underlaying code handle the default

Reply:
also update instantiateModelEntity the same way pls

Reply:
done